### ISSUE TITLE
t2 erase doesn't exit

### ISSUE THREAD

**Issue by Frijol** [tessel/t2-cli#210]
Output:

```sh
➜  pir git:(master) t2 erase
INFO Connecting to Tessel...
INFO Connected to Kelsey over LAN
INFO Erasing files from Flash...
INFO Files erased.

```

then it just hangs

**Comment by johnnyman727** [comment]
I've fixed this in a local branch. PR coming soon.

**Comment by johnnyman727** [comment]
Will be fixed in #215.
